CruCentralCoast/Database/DatabaseManager.swift

```diff
@@ -8,52 +8,379 @@
+        let realm = try! Realm()
+        // Get an always up-to-date list of realm objects of the given type
+        let communityGroups = realm.objects(CommunityGroup.self)
+        
+        // Ensure only one listener is created
+        if self.communityGroupListener == nil {
+            // Listen for Firebase updates on this collection
+            self.communityGroupListener = self.listenForChangesInCollection(CommunityGroup.self) {
+                // Attempt to call the appropriate callback on each listener
+                self.listeners.forEach { $0.value?.updatedCommunityGroups?() }
+            }
+        }
+        return communityGroups
+    }
+    
+    func getMinistryTeams() -> Results<MinistryTeam> {
+        // Get the default realm database
+        let realm = try! Realm()
+        // Get an always up-to-date list of realm objects of the given type
+        let ministryTeams = realm.objects(MinistryTeam.self)
+        
+        // Ensure only one listener is created
+        if self.ministryTeamListener == nil {
+            // Listen for Firebase updates on this collection
+            self.ministryTeamListener = self.listenForChangesInCollection(MinistryTeam.self) {
+                // Attempt to call the appropriate callback on each listener
+                self.listeners.forEach { $0.value?.updatedMinistryTeams?() }
+            }
+        }
+        return ministryTeams
+    }
+    
+    func getMissions() -> Results<Mission> {
+        // Get the default realm database
+        let realm = try! Realm()
+        // Get an always up-to-date list of realm objects of the given type
+        let missions = realm.objects(Mission.self)
+        
+        // Ensure only one listener is created
+        if self.missionListener == nil {
+            // Listen for Firebase updates on this collection
+            self.missionListener = self.listenForChangesInCollection(Mission.self) {
+                // Attempt to call the appropriate callback on each listener
+                self.listeners.forEach { $0.value?.updatedMissions?() }
+            }
+        }
+        return missions
+    }
+    
+    func getResources() -> Results<Resource> {
+        // Get the default realm database
+        let realm = try! Realm()
+        // Get an always up-to-date list of realm objects of the given type
+        let resources = realm.objects(Resource.self)
+        
+        // Ensure only one listener is created
+        if self.resourceListener == nil {
+            // Listen for Firebase updates on this collection
+            self.resourceListener = self.listenForChangesInCollection(Resource.self) {
+                // Attempt to call the appropriate callback on each listener
+                self.listeners.forEach { $0.value?.updatedResources?() }
+            }
+        }
+        return resources
+    }
+    
+//    func getResources(ofType type: ResourceType, _ completion: @escaping ([Resource])->Void) {
```

Comment:
Do we need this anymore?

Reply:
Nope! Good catch, I will remove it once I am done testing some final things